app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>here are all users</h1>
\ No newline at end of file
+<div id="container"> 
+  <% @users.each do |user|%>
+    <div id="container_container">
+      <img id="container_container_img" src="<%= user.Photo %>" alt="user profile">
+      <div id="container_container_div">
+        <h3><%= user.Name %></h3>
```

Comment:
- [ ] As requested in the project's requirements, kindly make sure that clicking on each user, routes to the user's show page(/users/:id). Hence we'll require a link. Hints

```
<a href ="/users/<%= user.id %>">
        <h3><%= user.Name %></h3>
 </a>
```


**link_to** helper can work best.